draft-ietf-tls-ticketrequests.md

```diff
@@ -90,27 +90,28 @@ be useful to use multiple, distinct tickets when opening parallel connections. C
 therefore bound the number of parallel connections they initiate by the number of tickets
 in their possession, or risk ticket re-use.
 - Connection racing: Happy Eyeballs V2 {{?RFC8305}} describes techniques for performing connection
-racing. The Transport Services Architecture implementation from {{?TAPS=I-D.ietf-taps-impl}} also describes how
-connections may race across interfaces and address families. In cases where clients have early
+racing. The Transport Services Architecture implementation from {{?TAPS=I-D.ietf-taps-impl}} also describes 
+how connections can race across interfaces and address families. In cases where clients have early
 data to send and want to minimize or avoid ticket re-use, unique tickets for each unique
 connection attempt are useful. Moreover, as some servers may implement single-use tickets (and even
 session ticket encryption keys), distinct tickets will be needed to prevent premature ticket
 invalidation by racing.
-- Connection priming: In some systems, connections may be primed or bootstrapped by a centralized
+- Connection priming: In some systems, connections can be primed or bootstrapped by a centralized
 service or daemon for faster connection establishment. Requesting tickets on demand allows such
 services to vend tickets to clients to use for accelerated handshakes with early data. (Note that
 if early data is not needed by these connections, this method SHOULD NOT be used. Fresh handshakes
 SHOULD be performed instead.)
 - Less ticket waste: Currently, TLS servers use application-specific, and often implementation-specific,
 logic to determine how many tickets to issue. By moving the burden of ticket count to clients,
-servers do not generate wasteful tickets for clients. Moreover, as ticket generation may involve
-expensive computation, e.g., public key cryptographic operations, avoiding waste is desirable.
-- Decline resumption: Clients may indicate they have no intention of resuming connections by
+servers do not generate wasteful tickets. As an example, clients might only request one ticket during
+resumption. Moreover, as ticket generation may involve expensive computation, e.g., public key 
```

Comment:
Looks like we're removing `may` words, but we have one here. How about `might`?

```suggestion
resumption. Moreover, as ticket generation might involve expensive computation, e.g., public key 
```

Reply:
Works for me!